py/gc.c

```diff
@@ -693,34 +693,6 @@ void gc_dump_alloc_table(void) {
         int c = ' ';
         switch (ATB_GET_KIND(bl)) {
             case AT_FREE: c = '.'; break;
-            /* this prints out if the object is reachable from BSS or STACK (for unix only)
```

Comment:
I would like to keep this for debugging purposes.  But, yes, would need to change the bss stuff to just scan the root pointer section of the mp_state_ctx structure.